packages/core/core/generate-text/run-tools-transformation.ts

```diff
@@ -0,0 +1,186 @@
+import { nanoid } from 'nanoid';
```

Comment:
Do we still have our custom nanoid config in `shared/utils`? Should this use that?